modules/reactor/include/meshgenerators/PolygonMeshGeneratorBase.h

```diff
@@ -65,18 +65,49 @@ class PolygonMeshGeneratorBase : public MeshGenerator
     HEXAGON_NUM_SIDES = 6
   };
 
+  /// A struct type used to contain multiple blocks's boundary layer related parameters
```

Comment:
```suggestion
  /// Contains multiple blocks's boundary layer related parameters
```

Reply:
is block the most descriptive word for them?
It's always a hexagonal ring region right?